langchain/src/retrievers/tests/vectara.int.test.ts

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable no-process-env */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { test, expect } from "@jest/globals";
+import { Document } from "../../document.js";
+import { VectaraRetriever } from "../vectara.js";
+import { VectaraStore } from "../../vectorstores/vectara.js";
+
+/**
+ * Steps to run this test:
+ * 1. Create a Vectara account at https://vectara.com/
+ * 2. Create a corpus with the following filter attributes: can_fly (level: document, data type: boolean), category (level: document, data type: integer)
```

Comment:
It's correct that you only need to create meta-data explicitly when creating the corpus for those attributes that you need to filter by. But as someone seeing it for the first time, I think it might be a bit confusing. Perhaps explain this, or just ask them to create all 4 attributes of meta-data (even if only 2 are used for filtering)?